database/storeprocds/spGetAllRole.sql

```diff
@@ -0,0 +1,26 @@
+-- =============================================
+-- Create basic stored procedure template
+-- =============================================
+
+-- Drop stored procedure if it already exists
+IF EXISTS (
+  SELECT * 
+    FROM INFORMATION_SCHEMA.ROUTINES 
+   WHERE SPECIFIC_SCHEMA = N'dbo'
+     AND SPECIFIC_NAME = N'spGetAllRole' 
+)
+   DROP PROCEDURE dbo.spGetAllRole
+GO
+
+CREATE PROCEDURE dbo.spGetAllRole
+	
+AS
+	SELECT RecId, Name, Description FROM ROLETABLE 
```

Comment:
Please change Name to [Name] because its sys reserved keyword

Reply:
Please change Description to [Description ] because its sys reserved keyword

Reply:
fixed